api/fixtures/example.go

```diff
@@ -472,12 +472,18 @@ func (o ExampleOptions) Resources() *ExampleResources {
 		cluster.Spec.DNS.BaseDomainPrefix = pointer.String(o.BaseDomainPrefix)
 	}
 
-	if o.ClusterCIDR != "" {
-		cluster.Spec.Networking.ClusterNetwork = []hyperv1.ClusterNetworkEntry{{CIDR: *ipnet.MustParseCIDR(o.ClusterCIDR)}}
+	var clusterNetworkEntries []hyperv1.ClusterNetworkEntry
+	for _, cidr := range o.ClusterCIDR {
```

Comment:
Is there a case where that requires more than two? If there isn't we should protect the API and the cli to prevent that from happening or fail appropriately.